src/MinimalDelegation.sol

```diff
@@ -51,7 +52,7 @@ contract MinimalDelegation is
 
     uint256 public packedEntrypoint;
 
-    function execute(bytes32 mode, bytes calldata executionData) external payable override {
+    function execute(bytes32 mode, bytes memory executionData) public payable override {
```

Comment:
oh interesting.. this essentially kills any kind of potential calldata decoding optimization. i think its fine but an alternative could be to dispatch internally

Reply:
the reason i changed it to memory was because when decoding batch of batches, it needs to call back into execute. so i think if we went back to calldata decoding, we would be able to switch it back to calldata then right?

Reply:
correct if we decoded in calldata or if we dispatched it internally

Reply:
yeah so we prob would want to call back to `execute` so it follows the erc